src/tasks/netcore/updateBlazorManifest.ts

```diff
@@ -0,0 +1,105 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import * as xml2js from 'xml2js';
+import ChildProcessProvider from "../../debugging/coreclr/ChildProcessProvider";
+import { DockerContainerVolume } from '../../debugging/coreclr/CliDockerClient';
+import LocalOSProvider from "../../debugging/coreclr/LocalOSProvider";
+import { OSTempFileProvider } from "../../debugging/coreclr/tempFileProvider";
+import { ext } from "../../extensionVariables";
+import { pathNormalize } from '../../utils/pathNormalize';
+import { PlatformOS } from '../../utils/platform';
+import { execAsync } from '../../utils/spawnAsync';
+import { DockerRunTaskDefinition } from "../DockerRunTaskProvider";
+import { DockerRunTaskContext } from "../TaskHelper";
+
+interface ContentRootAttributes {
+    BasePath: string;
+    Path: string;
+}
+
+interface ContentRoot {
+    $: ContentRootAttributes;
+}
+
+interface StaticWebAssets {
+    ContentRoot: ContentRoot[];
+}
+
+interface Manifest {
+    StaticWebAssets: StaticWebAssets;
+}
+
+export async function updateBlazorManifest(context: DockerRunTaskContext, runDefinition: DockerRunTaskDefinition): Promise<void> {
```

Comment:
Do we need to avoid running this for release-mode builds?

Reply:
We could do it only if the manifest path contains "debug" as that would be an accurate test the vast majority of the time.

Reply:
Edit: We can't actually do release builds the way VS does. The method we use for obtaining the output path (via `netCoreProjectProvider`) doesn't take inputs related to configuration, so it would always return the debug path.

So, this is moot.